scss/list.scss

```diff
@@ -0,0 +1,36 @@
+@import 'color';
+@import 'utils';
+
+.cards-container {
+  width: 100%;
+  max-width: 900px;
+  margin: 50px auto;
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+  .mini-card {
+    width: 100%;
+    max-width: 198px;
+    height: 300px;
+    margin-bottom: 36px;
+    padding: 28px;
+    @include box-shadow;
+
+    .title {
+      font-weight: bold;
+      font-size: 16px;
+      margin: 8px 0;
+    }
+
+    .author {
+      font-size: 14px;
+      color: $nevada-gray;
+    }
+
+    .mini-cover-image {
+      width: 100%;
+      max-width: 142px;
```

Comment:
why this max-width?

Reply:
I miss understood something listed in the tech guides